### PR TITLE
Fixed contents directive and alphabetized

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-1958](https://openedx.atlassian.net/browse/DOC-1958)

Completes two suggestions in the referenced story: provides a working contents list and alphabetizes the tables.

### Reviewers
- [x] Doc team review (sanity check): @catong @pdesjardins @srpearce 

### Testing
- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Squash commits

File: en_us/shared/exercises_tools/create_exercises_and_tools.rst [hunk 1]
```diff
@@ -223,18 +230,14 @@ General Exercises and Tools
...
-       course team and learners to work together to maintain the list of
-       resources. For example, team members and learners can suggest new
-       resources, vote for useful ones, or flag abuse and spam.
-     - Full support
 
 ********************************
 Image-Based Exercises and Tools
 ********************************
 
+Exercises and tools that involve an image file are listed alphabetically in
```
Comment (catong) [inline en_us/shared/exercises_tools/create_exercises_and_tools.rst]: I'd make this plural: "involve an image file" -> "involve image files"

File: en_us/shared/exercises_tools/create_exercises_and_tools.rst [hunk 2]
```diff
@@ -266,6 +269,9 @@ Image-Based Exercises and Tools
 Multiple Choice Exercises and Tools
 ************************************
 
+Exercises and tools that provide various ways for learners to select from
```
Comment (catong) [inline en_us/shared/exercises_tools/create_exercises_and_tools.rst]: Suggest striking unnecessary "various": "provide ways for learners". Reads a bit too convolutedly with "several options" later in the same sentence.

**Comment by catong** [comment]
Great! A couple of suggestions, otherwise :+1:

**Pull request merged**